docs/user-guide/command-line-interface.md

```diff
@@ -315,6 +317,22 @@ Example:
 
     eslint -v
 
+#### `--disableInlineConfig`
+
+This option prevents inline comments like `/*eslint-disable*/` or
+`/*global foo*/` from having any effect. This allows you to set an ESLint
+config without files modifying it. All inline config comments are ignored, e.g.:
+
+* `/*eslint-disable*/`
+* `/*eslint-enable*/`
+* `/*global*/`
+* `/*eslint*/`
+* `/*eslint-env*/`
```

Comment:
It should ignore those too (it prevents `modifyConfigsFromComments` from running at all), but I didn't document that or add tests for them. I'll do that and update the PR.